add-domain

```diff
@@ -352,3 +352,7 @@ soclaiemx.xyz
 coronavirus.zone
 bill-overdue.net
 dofuspoulesnoobs.fr
+alphaumi.com
+wasabiwallet.is
+wasabiwallet.app
```

Comment:
taken down

```
drill -T wasabiwallet.app
.       518400  IN      NS      a.root-servers.net.
.       518400  IN      NS      b.root-servers.net.
.       518400  IN      NS      c.root-servers.net.
.       518400  IN      NS      d.root-servers.net.
.       518400  IN      NS      e.root-servers.net.
.       518400  IN      NS      f.root-servers.net.
.       518400  IN      NS      g.root-servers.net.
.       518400  IN      NS      h.root-servers.net.
.       518400  IN      NS      i.root-servers.net.
.       518400  IN      NS      j.root-servers.net.
.       518400  IN      NS      k.root-servers.net.
.       518400  IN      NS      l.root-servers.net.
.       518400  IN      NS      m.root-servers.net.
app.    172800  IN      NS      ns-tld1.charlestonroadregistry.com.
app.    172800  IN      NS      ns-tld2.charlestonroadregistry.com.
app.    172800  IN      NS      ns-tld3.charlestonroadregistry.com.
app.    172800  IN      NS      ns-tld4.charlestonroadregistry.com.
app.    172800  IN      NS      ns-tld5.charlestonroadregistry.com.
app.    900     IN      SOA     ns-tld1.charlestonroadregistry.com. cloud-dns-hostmaster.google.com. 101427 21600 3600 259200 900
```

Reply:
@ninjacatcher do you remove this line so I can merge your MR

Reply:
@spirillen done, thanks